container_images/README.md

```diff
@@ -0,0 +1,110 @@
+# Docker container images
+
+The source for Docker container images used in our CI/CD infrastructure. Each
+folder represents one image. These are built by kaniko jobs in our Concourse
+`container-build` pipeline and generally published on gcr.io container registry,
+but are not made publicly available.
+
+We use these in [Prow] jobs and [Concourse] pipelines. Concourse pipelines are
+located in the [concourse](../concourse/) directory. Prow jobs are configured in
+the [oss-test-infra] repository.
+
+[Prow]: https://github.com/kubernetes/test-infra/tree/master/prow
+[Concourse]: https://concourse-ci.org
+[oss-test-infra]: https://github.com/GoogleCloudPlatform/oss-test-infra/tree/master/prow/prowjobs/GoogleCloudPlatform/gcp-guest/
+
+## Images
+
+A brief description of each container image is listed below.
+
+### build-essential
+
+This is a clone of a Debian container with the 'build-essential' package
+preinstalled. Usage: ??
```

Comment:
For this and other 'Usage: ??' locations -> please fill this in or remove for the merge in.

Reply:
no we want to keep that bc this is documenting state. some of these have no known usage